pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -276,6 +276,9 @@ type ScrapeConfigSpec struct {
 	// Authorization header to use on every scrape request.
 	// +optional
 	Authorization *v1.SafeAuthorization `json:"authorization,omitempty"`
+	// OAuth2 client credentials used to fetch a token for the targets.
```

Comment:
(nit) to be consistent with the other auth fields like `Authorization`.
```suggestion
	// OAuth2 configuration to use on every scrape request.
```